js/components/closed-caption-multi-audio-menu/multiAudioTab.js

```diff
@@ -0,0 +1,54 @@
+var React = require('react');
+var iso639 = require('iso-639-3');
+
+var Tab = require('./tab');
+var helpers = require('./helpers');
+var CONSTANTS = require('../../constants/constants');
+
+var MultiAudioTab = React.createClass({
+  render: function() {
+    var audioTracksList = this.props.list.map(
```

Comment:
Is this something we can do in componentWillUpdate/componentDidUpdate? So we don't recreate this list every time the component renders.

Reply:
Yeah it could be a good idea, 
but component gets unmounted after we close the menu, and lifecycle methods won't be called, because it's a new render every time we open the menu. We could work around it with static fields, but it'd create unnecessary complexity in checking for updates, so I think it's not worth it overall.

Reply:
Sounds good